create_category_result.php

```diff
@@ -0,0 +1,37 @@
+<?php include_once("create_category.php")?>
+<?php include_once("header.php")?>
+<?php include_once('database.php') ?>
+<?php include_once("database_functions.php")?>
+
+<div class="container my-5">
+
+<?php
+      
+      $categories = getCategoriesFromDatabase();
+      
+    if (isset($_POST["categorysubmit"])) {
+        $categoryTitle = $categoryDesc = "";
+
+    $categoryDesc = $_POST['categoryDesc'];
+    
+      if (empty($_POST["categoryTitle"])) {
+        $catErr = "Category Name is required";
+        echo "<script>alert('$categErr')</script>";
+      } else {
+        $categoryTitle = $_POST["categoryTitle"];
+      }
+    
+        foreach ($categories as $category) : 
```

Comment:
I suggest the comparison to be like this, so it'll always be an even comparison 
`.. if (strtolower($category['name']) == strtolower($categoryTitle)) { ..`